src/input.cpp

```diff
@@ -91,6 +94,7 @@ void create_rapi_json(JsonDocument &doc)
   doc["amp"] = evse.getAmps() * AMPS_SCALE_FACTOR;
   doc["voltage"] = evse.getVoltage() * VOLTS_SCALE_FACTOR;
   doc["pilot"] = evse.getChargeCurrent();
+  doc["max_current"] = evse.getMaxCurrent();
```

Comment:
This can be retrieved via the `/config` endpoint. Shouldn't be updated that often, most use cases should set the `max_current` via a claim.

Reply:
ok. I'll add the /config endpoint to mqtt to later. Do you think we should expose complete settable config too ? I'm wondering about security vs functionality

Reply:
That isn't what I meant, was just meaning this value can be retrieved by that HTTP endpoint and doesn't really fit in MQTT. Config updates are events via the `config_version` topic, when this changes the config can then be read via HTTP.

Reply:
Yes I got it later. I've removed it. Can you give a last check to see if it's readty to merge? I will prolly be busy next week.